app/src/Containers/Home/index.tsx

```diff
@@ -1,19 +1,25 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { BeautifulText, SexyContainer } from "./styles";
+import Header from "../Header";
+import JokeForm from "../JokeForm";
+import qs from "qs"
 
-const req = async () => {
+const reqJoke = async () => {
   const in_flight = await fetch("/api/joke/get");
   const output = await in_flight.json();
-  return output.joke;
+  return output.joke === undefined ? output.message : output.joke
 };
 
 export const HomePage = ({ joke }: { joke: string }) => {
   const is_joke_funny = joke === "penis";
   const [joke_remote, set_joke] = useState("");
-  debugger;
-  if (joke_remote === "") req().then((res) => set_joke(res));
+  const [token, setToken] = useState(qs.parse(document.cookie)['object Object'])
```

Comment:
So here is the higher most point in which the cookie is checked. I could pass it through on to header and so on and so forth. Another dilemma is trying to get this component to realize that document.cookie has been updated. I tried
 `useEffect(() => {
  setToken(qs.parse(document.cookie)['object Object'])
},[document.cookie])`